protocol/serialization-types/tests/test-serialization.rs

```diff
@@ -439,6 +449,7 @@ fn test_staged_ledger_diff_diff_commands_status() {
 #[wasm_bindgen_test]
 fn test_staged_ledger_diff_diff_coinbase() {
     block_path_test_batch! {
+        // FIXME: The given path is not valid for all test blocks
```

Comment:
This test makes invalid assumptions (that the variant is always the second one in 
```rust
pub enum CoinBase {
    Zero,
    // FIXME: other variants are not covered by current test block
    One(Option<CoinBaseFeeTransferV1>),
    Two,
}
```

This is invalid so we should just remove the test entirely. It is covered by the next line anyway

Reply:
Yeah, I think it's safe to disable the test for now since we have e2e coverage, ideally, the logic here should be, deserializing the binary into all variants and asserting there is exactly one being successful

Reply:
Check out https://github.com/ChainSafe/mina-rs/pull/218